platform-ajax/src/main/typescript/com/softicar/platform/ajax/engine/event/EventToClickDelegation.ts

```diff
@@ -0,0 +1,44 @@
+const EVENT_DELEGATIONS = new Map<HTMLElement, EventToClickDelegator>();
+
+function getOrCreateEventDelegator(element: HTMLElement) {
```

Comment:
We seem to randomly mix OOP and global functions. I think we should improve upon that in the future.

Reply:
planned for next PR